src/components/PageHeader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './PageHeader.css'
+import PropType from 'prop-types'
+import { isValidUrl } from '../utils/index'
+
+const PageHeader = props => {
+  const generateBackground = background => ({
+    background: isValidUrl(background) ? `url(${background})` : background
+  })
+
+  return (
+    <div className="jumbotron jumbotron-fluid jumbotron" style={generateBackground(props.background)}>
+      <div className="container">
+        <div className="row justify-content-center">
```

Comment:
not sure you need `justify-content-center` - i think you can make this `text-center`